### PR TITLE
Update DRAFT.md

### PR DESCRIPTION
**Pull request by nbx8**
Minor edits to draft.

### CODE REVIEW

File: DRAFT.md [hunk 1]
```diff
@@ -380,13 +380,13 @@ Copy constructors for an indirect value obtain an allocator by calling
 `allocator_traits<allocator_type>​::​select_on_container_copy_construction` on
 the allocator belonging to the indirect value being copied. Move constructors
 obtain an allocator by move construction from the allocator belonging to the
-container being moved. Such move construction of the allocator shall not exit
+container being moved. Such move construction of the allocator will not exit
```
Comment (jbcoe) [inline DRAFT.md:383]: This bit is standardese and needs to be 'shall'

File: DRAFT.md [hunk 2]
```diff
@@ -958,14 +958,14 @@ Copy constructors for a polymorphic value obtain an allocator by calling
 `allocator_traits<allocator_type>​::​select_on_container_copy_construction` on
 the allocator belonging to the polymorphic value being copied. Move constructors
 obtain an allocator by move construction from the allocator belonging to the
-container being moved. Such move construction of the allocator shall not exit
+container being moved. Such move construction of the allocator will not exit
```
Comment (jbcoe) [inline DRAFT.md:961]: This bit is standardese and needs to be 'shall'

**Review by jbcoe — CHANGES_REQUESTED** [review]

**Review by jbcoe — APPROVED** [review]

File: DRAFT.md (hunk 2 above)
Comment (jbcoe) [reply DRAFT.md]: I've amended the PR.

File: DRAFT.md (hunk 1 above)
Comment (nbx8) [reply DRAFT.md]: I did wonder. :)

**Pull request merged**